### ISSUE TITLE
Add Additional Information to Adobe Sign

### ISSUE THREAD

**Issue by tclarkCO** [cloud-elements/devportal#426]
## Description
* To perform `DELETE /envelopes` & `DELETE /agreements` the additional scope of `agreement_retention:account` is required. 
* We should add this to an 'Additional Information' section of the Adobe Sign documentation
* It should also be noted that this scope must be activated by Adobe Sign support
* If additional info is needed please see Tyler

**Comment by jimmacleodCE** [comment]
@tclarkCO @davewilks FYI, we had agreed to rename 'Additional Information' to 'Additional Tips' across the board so that it aligned better with creating an icon in the control console called 'Tips', so users know they can click that and get taken to this documentation.

**Comment by davewilks** [comment]
Makes sense @jimmacleodCE. I'll look through our backlog and see if that's captured. If not, I'll add an issue.
